src/Common/src/Interop/Ole32/Interop.PICTYPE.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+internal static partial class Interop
+{
+    internal static partial class Ole32
+    {
+        /// <summary>
+        ///  Picture type used with <see cref="IPicture"/> and
+        ///  <see href="https://docs.microsoft.com/en-us/windows/win32/com/pictype-constants"/>
+        /// </summary>
+        public enum PICTYPE : int
```

Comment:
Do we want to add a comment why this is `int`?
```suggestion
        /// <remarks>
        ///  This type is used with different field sizes, it is signed to prevent
        ///  sign-extension errors when going from smaller to larger field sizes.
        /// </remark>
        public enum PICTYPE : int
```